### PR TITLE
Fix tcp-chat-client(rust) to work

### PR DESCRIPTION
**Pull request by SeanKim**
The `examples/websocket/tcp-chat` crate has two binaries, one for the server and one for the client.
However, there is an error in the client-side code that prevents them from communicating with each other.
This PR corrected these errors and also removed the websocket communication related code that was not directly related to tcp-chat feature and made the client code bloated.

### CODE REVIEW

File: websockets/tcp-chat/src/main.rs [hunk 1]
```diff
@@ -228,26 +13,7 @@ async fn main() -> std::io::Result<()> {
...
-            // redirect to websocket.html
-            .service(web::resource("/").route(web::get().to(|| {
-                HttpResponse::Found()
-                    .header("LOCATION", "/static/websocket.html")
-                    .finish()
-            })))
-            // websocket
-            .service(web::resource("/ws/").to(chat_route))
-            // static resources
-            .service(fs::Files::new("/static/", "static/"))
```
Comment (JohnTitor) [inline websockets/tcp-chat/src/main.rs:248]: Since we don't serve `/static` now, we can remove it.

**Review by JohnTitor — COMMENTED** [review]
Featuring the TCP work (i.e. removing web client) is a good idea to me, to reduce duplication. We should also update README since it's now totally oudated.

File: websockets/tcp-chat/src/main.rs (hunk 1 above)
Comment (SeanKim) [reply websockets/tcp-chat/src/main.rs:248]: thanks for the review! i forgot that part I'll fix it as soon as I get home

File: websockets/tcp-chat/README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
-Added features:
+features:
 
-* Browser WebSocket client
-* Chat server runs in separate thread
-* Tcp listener runs in separate thread
+* Tcp Chat Server
+* Tcp Chat Client
 
 ## Server
```
Comment (JohnTitor) [inline websockets/tcp-chat/README.md:12]: `/name name` below should be removed as well.

**Comment by robjtede** [comment]
I updated this example to work in the v4 migration, if you still thing that this example should remove the websocket part then feel free to update the PR. I'll close this out after a week if there's been no updates.

**Pull request closed without merging**